Interni/Verbali Interni/2020_12_07/components/verbale.tex

```diff
@@ -0,0 +1,90 @@
+\section{Informazioni generali}
+
+\begin{itemize}
+
+	\item \textbf{Canale di comunicazione:} Zoom;
+	
+	\item \textbf{Data:} \DataMeeting{};
+	
+	\item \textbf{Ora inizio:} 21;
+	
+	\item \textbf{Ora fine:} 23;
+	
+	\item \textbf{Segretario:} Daniele Giachetto;
+	
+	\item \textbf{Partecipanti:}
+	
+		\begin{itemize}
+		
+			\item Daniele Giachetto;
+			\item Davide Albiero;
+			\item Francesco De Marchi;
+			\item Giosuè Calgaro;
+			\item Lucrezia Gradi;
+			\item Matteo Pagotto;
+			\item Tommaso Poppi.
+				 
+		\end{itemize}
+
+\end{itemize}
+
+\section{Ordine del giorno}
+
+\begin{itemize}
+
+	\item\textbf{Spiegazione struttura dei repository GitHub;}
+
+	\item\textbf{Raccolta delle domande da fare all'azienda per il capitolato 7;}
+
+	\item\textbf{Visualizzazione del documento \SdF{};}
+	
+	\item\textbf{Assegnazione del documento \PdP{}.}
+
+\end{itemize}
+
+\newpage
+
+
+\section{Resoconto}
+
+\subsection{Spiegazione struttura dei repository GitHub}
+
+È stato spiegato ad ogni membro del gruppo come verrà gestito il repository riguardante la documentazione. Sono state mostrate le pratiche che ogni componente del gruppo dovrà seguire per caricare i propri documenti e il modo con cui verrà effettuato il merge nel ramo main.
+
+\subsection{Raccolta delle domande da fare all'azienda per il capitolato 7}
+
+È stata scritta una mail contenente tutte le domande che ogni membro del gruppo voleva fare all'azienda riguardante il capitolato \textit{SSD: Soluzioni di sincronizzazione desktop}.
+La mail verrà spedita il giorno 2020-12-09.
+
+\subsection{Visualizzazione del documento \SdF{}}
+
+È stato mostrato il documento riguardante allo \SdF\ scritto da Davide e Giosuè. Daniele si è proposto di verificare lui stesso il documento una volta creata la Pull Request su GitHub.
+
+\subsection{Assegnazione del documento \PdP{}}
+
+A Tommaso e Matteo è stato assegnato la scrittura del \PdP .
+
+\newpage
+
+\section{Riepilogo delle decisioni \hfil}
+{
+	\setlength{\freewidth}{\dimexpr\textwidth-4\tabcolsep}
+	\renewcommand{\arraystretch}{1.5}
+	\setlength{\aboverulesep}{0pt}
+	\setlength{\belowrulesep}{0pt}
+	\rowcolors{2}{AzzurroGruppo!10}{white}
+	\begin{longtable}{L{.3\freewidth} L{.7\freewidth}}
+		\toprule 
+		\rowcolor{AzzurroGruppo!30}
+		\textbf{Codice} & \textbf{Decisione}\\
+		\toprule
+		\endhead
+		
+		VI\_\DataMeeting{}.1 & Tutti i componenti ora sanno come funziona GitHub. \\  
+		VI\_\DataMeeting{}.2 & Inviata la mail all'azienda. \\ 
+		VI\_\DataMeeting{}.3 & Daniele verificherà il documento. \\
+		VI\_\DataMeeting{}.4 & Assegnato il documento Piano di Progetto. \\  		
```

Comment:
Assegnata la stesura del documento, non il documento e basta. inoltre usa \PdP per scrivere Piano di Progetto (così è in italics automaticamente)